### PR TITLE
FA-76 removed forgot password from the project

### PR DESCRIPTION
**Pull request by HamdanSCode**
removed forgot password objects network objects ui objects etc..

### CODE REVIEW

**Comment by HamdanSCode** [comment]
realized I hadn't removed it from the create account and login pages fully as well as the viewmodel, it is now removed fully

File: app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepository.kt [hunk 1]
```diff
@@ -2,6 +2,6 @@ package com.hamdan.forzenbook.data.repository
 
 interface LoginRepository {
 
-    suspend fun getToken(email: String, password: String): LoginData?
+    suspend fun getToken(email: String): LoginData?
```
Comment (ForzenAcademy) [inline app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepository.kt:5]: What is this returning?

File: app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt [hunk 2]
```diff
@@ -42,13 +42,11 @@ fun CreateAccountContent(
                 state.lastName,
```
Comment (ForzenAcademy) [inline app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt]: Replace a bunch of `state.` with `.apply` or whatnot

**Review by ForzenAcademy — APPROVED** [review]

File: app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepository.kt (hunk 1 above)
Comment (HamdanSCode) [reply app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepository.kt:5]: well its giving back a data class that just holds a string which is currently the token, obviously I have to make a second function for the get and post as the post is the one that actually gets a response and it needs both the email and a validation code but thats out of scope of this ticket, I will follow up on this in another ticket

Comment (HamdanSCode) [reply app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepository.kt:5]: so long story short it should return nothing for that one and be renamed

**Pull request merged**